BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -156,17 +157,25 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
       guard let host = state.url.host else { return }
       attentionView.valueLabel.text = "0%"
       
-      state.ledger.publisherInfo(forId: host) { info in
-        guard let publisher = info else { return }
-        assert(Thread.isMainThread)
-        publisherView.setVerified(publisher.verified)
+      publisherView.onCheckAgainTapped = { [weak self] in
+        guard let self = self else { return }
         
-        if let percent = self.state.ledger.currentActivityInfo(withPublisherId: publisher.id)?.percent {
-          attentionView.valueLabel.text = "\(percent)%"
+        publisherView.checkAgainButton.isLoading = true
+        self.state.ledger.publisherInfo(forId: host) { info in
```

Comment:
Hah, this is actually not the call that has to be made here. Its an API called `RefreshPublisher` which we haven't wrapped yet in `BATBraveLedger`, so I'll get you to update this when that API is available

Reply:
Fixed.